inc/class-upgrade.php

```diff
@@ -780,6 +788,24 @@ private function upgrade_141() {
 		$wpdb->show_errors = $show_errors;
 	}
 
+	/**
+	 * Resets the permalinks of attachments to `null` in the indexable table for the upgrade routine to 14.1.
+	 */
+	private function reset_permalinks_of_attachments_for_141() {
+		global $wpdb;
+
+		// If migrations haven't been completed succesfully the following may give false errors. So suppress them.
+		$show_errors       = $wpdb->show_errors;
+		$wpdb->show_errors = false;
+
+		// Reset the permalinks of the attachments in the indexable table.
+		$indexable_table = Model::get_table_name( 'Indexable' );
+		$query           = "UPDATE $indexable_table SET permalink = NULL WHERE object_type = 'post' AND object_sub_type = 'attachment'";
```

Comment:
This query seems duplicated. It should probably be removed from the `clean_up_private_taxonomies_for_141` function.